proposal/color-4-new-spaces.md

```diff
@@ -0,0 +1,1645 @@
+> out-of-gamut for the [color space][] used. Similarly, this color-channel
+> inspection function may return out-of-gamut channel values.
+
+* ```
+  channel($color, $channel, $space)
+  ```
+
+  * If `$space` is null:
+
+    * Let `space` be the result of calling `space($color)`, and let `color` be
+      the value of `$color`.
+
+  * Otherwise:
+
+    * Let `color` be the result of calling `to-space($color, $space)`, and let
+      `space` be the value of `$space`.
+
+  * If `space` is a known [color space][], let `channels` be a map of channel
+    names defined for `space`, and their corresponding values in `color`, or a
+    map with 1-indexed number keys and their corresponding values in `color`
+    otherwise.
+
+  * Let `value` be the result of calling `map.get(channels, $channel)`.
+
+  * If `value` is `null`, throw an error.
+
+  * Otherwise, return `value`.
+
+
+## Modified Color Module Functions
+
+### `color.hwb()`
+
+These functions are now deprecated. Authors should use global `hwb()` instead.
+
+> Channel clamping and scaling have been removed from the global function,
+> since we now allow out-of-gamut color-channels to be stored as specified.
+
+* ```
+  hwb($channels)
+  ```
+
+  This function is available as a global function named `hwb()`.
+
+* ```
+  hwb($hue, $whiteness, $blackness, $alpha: 1)
+  ```
+
+  * Return the result of calling the global function
+    `hwb($hue $whiteness $blackness / $alpha)`.
+
+### `color.mix()`
+
+```
+mix($color1, $color2,
+  $weight: 50%,
+  $method: null)
+```
+
+* If `$method` undefined:
+
+  * If `$color1` and `$color2` are both legacy colors, let `method` be `rgb`.
+
+  * Otherwise, throw an error.
+
+    > Method is required for non-legacy colors. This matches the `color-mix()`
+    > function defined in [Colors Level 5][color-5], and allows us to add
+    > additional default behavior in the future.
+
+* Otherwise:
+
+  * If `method` is not a [color interpolation method][color-method], throw an
+    error.
+
+  * Let `method` be the value of `$method`.
+
+  * Let `space` be the [color space][] specified in `method`.
+
+* If `space` is one of 'hsl' or 'hwb':
+
+  * Let `color1` and `color2` be the result of [converting][] and
+    [gamut mapping][] `$color1` and `$color2` respectively into `space`.
+
+  > These color spaces are unable to express colors outside the `srgb` gamut.
+
+* Otherwise, let `color1` and `color2` be the result of [converting][]
+  `$color1` and `$color2` respectively into `space`.
+
+* Return the result of [interpolating](#interpolating-colors) between `color1`
+  and `color2` with the specified `$weight` and `method`.
+
+### `color.alpha()`
+
+* ```
+  alpha($color)
+  ```
+
+  * If `$color` is not a color, call the other overload and return its result.
+
+  * Return the alpha channel of `$color` as a unitless number.
+
+* ```
+  alpha($args...)
+  ```
+
+  > This overload exists to support Microsoft's proprietary [`alpha()`
+  > function][].
```

Comment:
shouldn't we deprecate that overload in the namespaced function ?

Reply:
For now, I've removed this entire `alpha()` section based on Natalie's comments above, since it doesn't make any changes. Not sure if we want to add a special mention of this overload in the deprecations section?